docs/traefik.md

```diff
@@ -0,0 +1,104 @@
+---
+id: traefik
+title: traefik reverse proxy
+---
+
+# Traefik static config file
+
+/etc/traefik/traefik.yml
+```yaml
+api:
+  dashboard: true
+  insecure: true
```

Comment:
I'm okay with having a traefik guide on the website, but I'd want this to be a generic one, so nothing cloudflare specific. With http acme most of the settings aren't needed as it's the default.

Reply:
I'll fix this later